backend/src/main/java/com/example/backend/entity/Stop.java

```diff
@@ -18,6 +20,10 @@ public String getTag() {
         return tag;
     }
 
+    public String getName() {
+        return name;
+    }
+
     public Location getLocation() { return location; }
 
     public HashSet<String> getRoutes() {
```

Comment:
One unrelated thing I noticed. This should be getRouteTags

Reply:
Fixed